### ISSUE TITLE
Cannot update translations through the website form

### ISSUE THREAD

**Issue by zeecho** [thewca/worldcubeassociation.org#7210]
When trying to update the esperanto translation of the website (https://www.worldcubeassociation.org/translations/edit), I got the good old "We're sorry but something went wrong" after submitting the form.
As everything on the website end seems to work while testing locally (Github posting deactivated), I think the issue might be because of expired Github credentials?

**Comment by gregorbg** [comment]
I thought that this was related to the recent Rails 7 update, but indeed it was a problem with the GitHub token. So thanks for pointing me in the right direction :)

For some reason, GitHub arbitrarily changed the amount of permissions (aka scopes) that the access token needs in order to be able to create a new branch. They did not announce this to us and it was not documented anywhere (at least not on their API documentation pages) and I basically figured this out by trial-and-error'ing.

To the devs: This was solved by adding the `repo:status` scope to our PAT. No idea why this is suddenly required now when it worked for months before.

**Comment by gregorbg** [comment]
TL;DR It's fixed :wink:

**Comment by zeecho** [comment]
Perfect! Thanks for having quickly fixed this :)
